### DISCUSSION TITLE
Current docker setup in `next.js` repo doesn't work with github codespaces

### DISCUSSION THREAD

**Discussion by jankaifer** [vercel/next.js#43451] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### What browser are you using? (if relevant)

Brave

### Describe the Bug

Codespace is running in recovery mode.

### Expected Behavior

It should "just work".

### To Reproduce

Try to open a codespace in this repo.

**Comment by jankaifer** [comment]
original report: https://github.com/vercel/next.js/pull/43138#issuecomment-1328026282

**Reply by akshitsinha** [reply]
Ref #44157
